src/components/Calendar/index.js

```diff
@@ -484,7 +497,12 @@ class Calendar extends PureComponent {
               isVertical ? this.styles.monthsVertical : this.styles.monthsHorizontal
             )}>
             {new Array(this.props.months).fill(null).map((_, i) => {
-              const monthStep = addMonths(this.state.focusedDate, i);
+              let monthStep;
+              if (this.props.calendarFocus === 'forwards') {
+                monthStep = addMonths(this.state.focusedDate, i);
+              } else {
+                monthStep = subMonths(this.state.focusedDate, this.props.months - 1 - i);
+              }
```

Comment:
let's only have an `if` statement to override the value if calendarFocus is `backwards`.
More granule change would help us avoid possible bugs imho.

```suggestion
              let monthStep = addMonths(this.state.focusedDate, i);;
              if (this.props.calendarFocus === 'backwards') {
                monthStep = subMonths(this.state.focusedDate, this.props.months - 1 - i);
              }
```